src/test/java/stepdefs/SearchStepDefs.java

```diff
@@ -35,19 +36,47 @@ public void iAmOnTheGoogleUkHomepage() {
     driver.manage().window().maximize();
     driver.get("https://www.google.co.uk");
   }
-  
-  @When("I enter a search term")
-  public void iEnterASearchTerm() {
+
+  @Given("^I am on the Google UK$")
+  public void iAmOnTheGoogleUk() {
+    driver.manage().window().maximize();
+    driver.get("https://www.google.co.uk");
+  }
+  @When("I enter {string} search term")
+  public void iEnterSearchTerm(String searchItem) {
+
     WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(15));
-    wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//button[@id='L2AGLb2']"))).click();
-    driver.findElement(By.xpath("//input[@title='Search']")).sendKeys("BBC news");
-    driver.findElement(By.xpath("//input[@title='Search']")).sendKeys(Keys.ENTER);
+    GoogleHomePage.CookiesButton_search(driver).click();
+    GoogleHomePage.textBox_search(driver).sendKeys(searchItem);
+    //wait.until(ExpectedConditions.elementToBeClickable(By.name("btnK"))).sendKeys(Keys.ENTER);
+    wait.until(ExpectedConditions.elementToBeClickable(GoogleHomePage.GoogleButton_search(driver))).sendKeys(Keys.ENTER);
+    GoogleHomePage.GoogleButton_search(driver).click();
```

Comment:
I think you can remove this mate as your ENTER press handles it

Reply:
Removed the line:
  @When("I enter {string} search term")
  public void iEnterSearchTerm(String searchItem) {

    WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(15));
    GoogleHomePage.CookiesButton_search(driver).click();
    GoogleHomePage.textBox_search(driver).sendKeys(searchItem);
    wait.until(ExpectedConditions.elementToBeClickable(GoogleHomePage.GoogleButton_search(driver))).sendKeys(Keys.ENTER);
  }